src/main/java/dev/pgm/community/polls/commands/PollManagementCommands.java

```diff
@@ -0,0 +1,140 @@
+
+  @CommandMethod("")
+  public void info(CommandAudience audience) {
+    if (polls.isRunning()) {
+      polls.sendPollDetails(audience);
+    } else {
+      polls.sendBuilderDetails(audience);
+    }
+  }
+
+  @CommandMethod("start [duration]")
+  public void start(
+      CommandAudience audience,
+      @Argument("duration") Duration duration,
+      @Flag(value = "delay", aliases = "d") Duration delay) {
+    if (polls.isDelayedStartScheduled()) {
+      audience.sendWarning(
+          text("A poll is already scheduled to start soon!")
+              .hoverEvent(
+                  HoverEvent.showText(
+                      text()
+                          .append(text("Use "))
+                          .append(text("/poll end", NamedTextColor.AQUA))
+                          .append(text(" to cancel delayed start."))
+                          .color(NamedTextColor.GRAY)
+                          .build())));
+      return;
+    }
+
+    if (!polls.canStart()) {
+      audience.sendWarning(
+          text()
+              .append(
+                  text("Your poll is not ready to start! Check ")
+                      .append(text("/poll", NamedTextColor.AQUA))
+                      .append(text(" for more info.")))
+              .hoverEvent(HoverEvent.showText(text("Click to view poll info")))
+              .clickEvent(ClickEvent.runCommand("/poll"))
+              .build());
+      return;
+    }
+
+    if (duration != null) {
+      polls.getBuilder().duration(audience, duration);
+    }
+
+    polls.getBuilder().creator(audience.getId().orElse(null));
+    if (delay == null) {
+      polls.start(audience);
+    } else {
+      polls.delayedStart(audience, delay);
+    }
+  }
+
+  @CommandMethod("end")
+  public void end(CommandAudience audience) {
+    polls.end(audience);
+  }
+
+  @CommandMethod("question [question]")
+  public void question(CommandAudience audience, @Argument("question") @Greedy String question) {
+    checkPoll();
+    polls.getBuilder().question(audience, question);
+  }
+
+  @CommandMethod("duration [duration]")
+  public void timelimit(CommandAudience audience, @Argument("duration") Duration duration) {
+    checkPoll();
+    polls.getBuilder().duration(audience, duration);
+  }
+
+  @CommandMethod("map [map]")
+  public void map(CommandAudience audience, @Argument("map") MapInfo map) {
+    checkPoll();
+    polls.getBuilder().map(audience, map);
+  }
+
+  @CommandMethod("kick [target]")
+  public void kickPlayer(CommandAudience audience, @Argument("target") Player target) {
+    checkPoll();
+    polls.getBuilder().kickPlayer(audience, target);
+  }
+
+  @CommandMethod("command [command]")
+  public void command(CommandAudience audience, @Argument("command") @Greedy String command) {
+    checkPoll();
+    polls.getBuilder().command(audience, command);
+  }
+
+  @CommandMethod("mutation [mutation]")
+  public void mutation(CommandAudience audience, @Argument("mutation") MutationType mutation) {
```

Comment:
Presumably, this should check the mutation permission here, as someone could conceivably have poll perms but not mutation perms, which would let them make a confusing poll to add a mutation, which wouldn't actually do so.